src/ui/Island/Cursor/CursorLabel.tsx

```diff
@@ -15,7 +20,12 @@ export default function CursorLabel({
 }: CursorLabelProps) {
   if (!cursor) return null
 
-  const { name, realmIcon, stakingRealmColor, cursorTextColor } = userInfo
+  const { name, realmName, stakingRealmColor, cursorTextColor } = userInfo
+
+  const realmIcons = { arbitrum, cyberconnect, gitcoin, galxe, frax }
```

Comment:
Should be stored in consts folder together with other constant realm data I think